src/lib/kdb/kdb_default.c

```diff
@@ -287,7 +287,8 @@ krb5_db_def_fetch_mkey_stash(krb5_context   context,
         goto errout;
     }
 
-    if (!(key->contents = (krb5_octet *)malloc(key->length))) {
+    if (!(key->contents = (krb5_octet *)malloc(
+              sizeof(key->contents[0])*key->length))) {
```

Comment:
The more I think about this, the more I'm confused about why Coverity triggers on it for Martin (and not on any other code that allocates a uint8_t buffer).  key->contents is of type krb5_octet *, and krb5_octet is a typedef for uint8_t, and uint8_t is a typedef for unsigned char with glibc (looking at stdint.h).  So why does Coverity think we need to multiply?